style.css

```diff
@@ -0,0 +1,4 @@
+body {
+  padding: auto;
+  margin: auto;
+}
```

Comment:
🌵 🌵 It is better to add more style to make your web page looks smart and attractive 🌵 🌵